MaterialComponentsExamples.podspec

```diff
@@ -1,18 +1,19 @@
 experimental_sources = [
   'components/*/examples/experimental/supplemental/*.{h,m,swift}',
   'components/*/examples/experimental/*.{h,m,swift}',
+  'components/schemes/*/examples/src/*.{h,m,swift}',
 ]
 
 experimental_headers = [
   'components/*/examples/experimental/supplemental/*.h',
   'components/*/examples/experimental/*.h',
+  'components/schemes/*/examples/src/*.h',
```

Comment:
Shouldn't this be something like `/schemes/*/examples/experimental/*.h`?  I don't think we want ALL examples to be experimental, do we?